Changelog.md

```diff
@@ -24,7 +28,8 @@ Compiler Features:
  * SMTChecker: Properties that are proved safe are now reported explicitly at the end of analysis. By default, only the number of safe properties is shown. The CLI option ``--model-checker-show-proved-safe`` and the JSON option ``settings.modelChecker.showProvedSafe`` can be enabled to show the full list of safe properties.
  * Standard JSON Interface: Add experimental support for importing ASTs via Standard JSON.
  * Yul EVM Code Transform: If available, use ``push0`` instead of ``codesize`` to produce an arbitrary value on stack in order to create equal stack heights between branches.
-
+ * Commandline Interface: Add ``--ast-compact-json`` option with Yul input (e.g. with ``--strict-assembly``, ``--yul``)
+ * Commandline Interface: Add ``--ir-ast-json`` and ``--ir-optimized-ast-json`` output options which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
```

Comment:
```suggestion
```

Reply:
Could you explain the reason for this suggestion?

Reply:
This is the `0.8.20` section if I'm not mistaken?

Reply:
Oh sorry! Thought I had deleted that haha